d2l-table-col-sort.js

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
This was once referenced from the LMS, but that usage has been removed so deleting this file.